### PR TITLE
Update readme with required scopes for GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by bingneef**
Love the functionality!

It was unclear what scopes the token required, but I believe this should do the trick.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
 # Workflow run cleanup action
 
-This action cleans up previously running instances of a workflow
-on the same branch. This accomplishes the task of automatically
-cancelling CI runs on pushes to the same branch, which is a common
-feature in most CI systems but currently not possible with
-GitHub actions.
+This action cleans up previously running instances of a workflow on the same branch. This accomplishes the task of automatically cancelling CI runs on pushes to the same branch, which is a common feature in most CI systems but currently not possible with GitHub actions.
```
Comment (rokroskar) [inline README.md]: I don't think anything changed here? Could you undo the change that puts all the text on one line?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
 
 ## Configuration
 
-None. The action uses the GitHub action environment variables
-to obtain the workflow name and branch. You must, however,
-set the `GITHUB_TOKEN` environment variable:
+None. The action uses the GitHub action environment variables to obtain the workflow name and branch. You must, however, create an [access token](https://github.com/settings/tokens) that includes the scope `repo`. Then, you can set the `GITHUB_TOKEN` environment variable:
```
Comment (rokroskar) [inline README.md]: You don't strictly need to create a new access token - one is already provided for you automatically in github actions (see [here](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret)). Maybe it would be better to word this differently, so it doesn't sound like you *have* to create a personal access token?

File: README.md [hunk 3]
```diff
@@ -20,9 +14,7 @@ env:
   GITHUB_TOKEN: ${{ secret.GITHUB_TOKEN }}
 ```
 
-You may want to disable this action from running on tags or master,
-especially if you have CD pipelines linked to your CI passing on
-every commit. In that case, something like this should work:
+You may want to disable this action from running on tags or master, especially if you have CD pipelines linked to your CI passing on every commit. In that case, something like this should work:
```
Comment (rokroskar) [inline README.md]: I don't think any text changed here? Please undo the formatting change.

**Review by rokroskar — CHANGES_REQUESTED** [review]
Thanks a lot for the suggestion! I think the wording could be a bit more accurate if it didn't sound like creating a custom personal access token is necessary - the action will work fine with the generic `GITHUB_TOKEN` that is provided by default by Github actions, but the variable still needs to be set.

**Comment by bingneef** [comment]
You were right, I now included it as a note (saying that if a personal access token is used, that scope must be included). Hope you agree 🎉

**Review by rokroskar — APPROVED** [review]
Looks good, thanks!!!
